Microsoft.Dynamics365.UIAutomation.Sample/UCI/TestsBase.cs

```diff
@@ -1,8 +1,14 @@
-﻿using System;
+﻿// Enable System.Diagnostics Trace/Debug
+#define DEBUG 
+#define TRACE
+
```

Comment:
Require to enable traces, can be anywhere in the assembly. See [documentation](https://docs.microsoft.com/en-us/dotnet/framework/debug-trace-profile/tracing-and-instrumenting-applications) for more infos